kdaglas/nomaCakes-UI
# Create project structure of the app using Angular

### What does this PR do?
 This PR is to set up the nomaCakes UI project from scratch using angular CLI

### Description of Task to be completed?
 Use Angular 8 and set up the project and create the necessary folders required for the components

### How should this be manually tested?
- Fetch this branch by running the command `git fetch origin ch-setup-project` in the terminal
- Check into this branch by running `git checkout ch-setup-project` in the terminal
- Start the application by running `npm start`
